jsk_fetch_robot/README.md

```diff
@@ -119,7 +119,32 @@ fetch-interface function APIs
   ```
 (send *ri* :stop-grasp)
   ```
+  
+gazebo内で、逆運動学でfetchを動かす。
```

Comment:
@708yamaguchi Hi, thank you for great contribution. Could you please re-write explanation in English?